Sources/Datadog/RUM/Instrumentation/Resources/URLSessionRUMResourcesHandler.swift

```diff
@@ -146,13 +146,14 @@ extension DistributedTracing {
         let spanID = traceIDGenerator.generate()
 
         var request = request
+        // To make sure the generated traces from RUM don’t affect APM Index Spans counts.
+        request.setValue("rum", forHTTPHeaderField: TracingHTTPHeaders.originField)
+
         headerTypes.forEach {
             let writer: TracePropagationHeadersWriter
             switch $0 {
             case .datadog:
                 writer = HTTPHeadersWriter(sampler: sampler)
-                // To make sure the generated traces from RUM don’t affect APM Index Spans counts.
-                request.setValue("rum", forHTTPHeaderField: "x-datadog-origin")
```

Comment:
question/ Have we consulted this change with other teams? I'm referring to recent Slack discussion on setting DD-specific header in OTel-instrumented requests.